core/templates/pages/topic-viewer-page/topic-viewer-page.component.css

```
@@ -7,9 +7,151 @@
+  border: 0;
+  border-radius: 0;
+  box-shadow: none;
+  margin: 0;
+  max-width: 100%;
+  padding: 0;
+  width: 100%;
+}
+
+topic-viewer-page .redesigned-studyguide-section subtopics-list .tab-title {
+  color: #00645c;
+  font-family: 'Capriola', 'Roboto', Arial, sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Capriola" (font-family-name-quotes)

(font-family-name-quotes)